src/community/wps-download/src/main/java/org/geoserver/wps/gs/download/RasterDownload.java

```diff
@@ -236,6 +240,29 @@ public Resource execute(String mimeType, final ProgressListener progressListener
                 throw new WPSException("The reader did not return any data for current input "
                         + "parameters. It normally means there is nothing there, or the data got filtered out by the ROI or filter");
             }
+            
+            //
+            // STEP 0 - Check for bands, select only those specified
+            //
+            
+            if (bandIndices!=null && bandIndices.length>0){                
```

Comment:
Sorry about that. I believe it was fixed (among some other details) in [d649b8f](https://github.com/mantesat/geoserver/commit/d649b8f29bc9413aab7c3328bfcbdae414b3a9fa). 

I was confused because there was already existing code exceeding the 100 char line limit...